packages/devextreme/js/ui/context_menu/ui.context_menu.js

```diff
@@ -673,6 +677,18 @@ class ContextMenu extends MenuBase {
         return availableHeight - SUBMENU_PADDING;
     }
 
+    _dimensionChanged() {
+        $(`.${DX_MENU_ITEM_EXPANDED_CLASS}`).each((_, item) => {
```

Comment:
[refactoring, optimization]
isn't it's better to loop through `this._shownSubmenus`?